app/views/single_pages/_assay.html.erb

```diff
@@ -0,0 +1,15 @@
+    <div class="simpleForm">
+        <ul class="nav nav-tabs">
+            <li class="active "><a data-toggle="tab" href="#assay-details"><i class="glyphicon glyphicon-list-alt"></i>
+                    Details</a></li>
+             <li>
+                <a data-toggle="tab" href="#assay-related-items">
+                    <i class="glyphicon glyphicon-th-list"></i>&nbsp;Related items
+                </a>
+            </li>
+        </ul>
+        <div class="tab-content">
+            <div id="assay-details" class="tab-pane fade in active" style="padding:10px"></div>
+            <div id="assay-related-items" class="tab-pane fade" style="padding:10px"></div>
+        </div>
+    </div>
```

Comment:
these files are all very similar with only a couple of ids or hrefs based on the resource type. Could be put into a generic partial that derives the naming form the resource itself or is passed in as a local attribute. It would make it easier to maintain and remove the need to add a new page whenever a new resource type is added